x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/topn/TopNOperator.java

```diff
@@ -346,7 +346,8 @@ public TopNOperator(
         this.elementTypes = elementTypes;
         this.encoders = encoders;
         this.sortOrders = sortOrders;
-        this.inputQueue = new Queue(topCount);
+        breaker.addEstimateBytesAndMaybeBreak(Queue.sizeOf(topCount), "esql engine topn");
```

Comment:
nit: maybe move this memory acquire to Queue ctor for consistency with Queue#close.

Reply:
I'll leave a comment about why I'm not doing that! The trouble is that we allocate the memory in the `super`. I guess I can make the ctor private and call a static. that's good. I'll do that.